modules/git/repo_commit.go

```diff
@@ -204,13 +204,26 @@ func (repo *Repository) FileChangedBetweenCommits(filename, id1, id2 string) (bo
 }
 
 // FileCommitsCount return the number of files at a revision
-func (repo *Repository) FileCommitsCount(revision, file string) (int64, error) {
-	return CommitsCountFiles(repo.Ctx, repo.Path, []string{revision}, []string{file})
+func (repo *Repository) FileCommitsCount(revision, file, not string) (int64, error) {
+	return CommitsCount(repo.Ctx,
+		CommitsCountOptions{
+			RepoPath: repo.Path,
+			Not:      not,
+			Revision: []string{revision},
+			RelPath:  []string{file},
+		})
+}
```

Comment:
There is only one call to `FileCommitsCount(..., not)`, other calls are all `FileCommitsCount(..., "")`

I think that only `FileCommitsCount(..., not)` could be changed to `CommitsCount(repo.Ctx, ...)`, then the other `FileCommitsCount(..., "")` could be reverted to ``FileCommitsCount(...)` to avoid unnecessary changes.

Otherwise oiverall L-G-T-M.